### ISSUE TITLE
Mulighed for at anvende LIFAOIS opslag ved "Vis ejendomsoplysninger"?

### ISSUE THREAD

**Issue by ChristianHelringAndersen** [Septima/spatialsuite-s4#302]
Hej Klavs

Vi har LOIS og dermed også adgang til det webbaserede opslagsværktøj (webois) som LIFA stiller til rådighed. Kunne man forestille sig at man istedet for at anvende OIS.dk ved ejendomsopslag kunne anvende denne løsning (eventuelt styret af en parameter)?

Linket til en ejendom er:

https://webois.lifa.dk/OIS/default.aspx?UID=XXX&PWD=YYYY&komnr=175&ejdnr=ZZZZ

**Comment by klavspc** [comment]
Ja, det kan man sagtens og det ville ikke være svært at lave. Det lyder
lidt som de forskellige plan-tools man kan lægge på som enten linker til
plansystemet eller til Odeum/PlanDK.

LIFA udbyder allerede i dag nogle moduler til SpS (ejdExplorer og
tlExplorer), som, udover at give menuer og knapper i arbejdsgeometri, også
lægger links ind i s4. Er der andre funktioner i lifa's webois som man
kunne forestille sig skal integreres i SpS? I så fald kan det være, at det
er et modul som lifa bør udbyde.


*Klavs Pihlkjær Christensen*
Partner, seniorkonsulent
+45 9132 6946
klavs@septima.dk
--------------------------
Septima P/S
Frederiksberggade 19, 2. sal
1459 Kbh K
www.septima.dk


Den ons. 20. feb. 2019 kl. 09.51 skrev Christian Helring Andersen <
notifications@github.com>:

**Comment by ChristianHelringAndersen** [comment]
Hej Klavs

nej ikke umddelbart. Det er en ren frontend for OIS data stylet lidt pænere end OIS ;-) (med lidt mere data)

**Comment by klavspc** [comment]
@ ChristianHelringAndersen : 

Er dette, hvad du kunne ønske dig: Et tool, der for matrikelnummer linker til
https://webois.lifa.dk/OIS/default.aspx?UID=XXX&PWD=YYYY&komnr=175&ejdnr=ZZZZ, hvor

1. UID, PWD, komnr er parametre og
2. ejdnr er esr

**Comment by ChristianHelringAndersen** [comment]
Hej Klavs

præcis.

DBH

Christian

Den ons. 11. sep. 2019 kl. 11.29 skrev Klavs Pihlkjær <
notifications@github.com>:
